docs/reference/ccr/requirements.asciidoc

```diff
@@ -42,4 +42,25 @@ The number of soft deletes to retain. Soft deletes are collected during merges
 on the underlying Lucene index yet retained up to the number of operations
 configured by this setting. The default value is `0`.
 
-For more information about index settings, see {ref}/index-modules.html[Index modules].
\ No newline at end of file
+For more information about index settings, see {ref}/index-modules.html[Index modules].
+
+[float]
+[[ccr-overview-beats]]
+==== Setting soft deletes on indices created by APM Server or Beats
+
+If you want to replicate indices created by APM Server or Beats, and are
+allowing APM Server or Beats to manage index templates you will want to add some
```

Comment:
I would state this more directly and say:

If you want to replicate indices created by APM Server or Beats, and are
allowing APM Server or Beats to manage index templates, you need to
enable soft deletes. To enable soft deletes, add the following settings
to the APM Server or Beats configuration file.